frontend/src/components/dashboard/MineSearch.js

```diff
@@ -19,11 +21,14 @@ const propTypes = {
   searchValue: PropTypes.string,
 };
 
-const defaultProps = {
-  mineNameList: [],
-};
+const checkAdvancedSearch = ({ status, region, tenure, commodity, TSF, major }) =>
+  TSF || major || some([status, region, tenure, commodity], negate(isEmpty));
```

Comment:
clever

Reply:
Bad clever? This should be fairly standard for a project using Lodash (or a similar library)

Reply:
no, good clever.